jsx/Form.js

```diff
@@ -1062,15 +1062,19 @@ class DateElement extends Component {
       maxYear = '9999';
     }
 
+    const currentDate = new Date();
+    const currentDay = ('0' + currentDate.getDate()).slice(-2);
+    const currentMonth = ('0' + (currentDate.getMonth() + 1)).slice(-2);
```

Comment:
What are you trying to do with this '0' hack? What happens in November or December? Or day of the month >= 10?

Reply:
@driusan it keeps a '0' if single digit response for the day or month. So September would be 09 instead of 9. Similar for the day of the month. If the day is 7 the currentDay will be 07. If the day is 11 the currentDay will be 11.

Reply:
@maltheism i think it would work but could you please add a comment line in the code describing that it's needed because getmonth and getdate return values need to be padded before saving

Reply:
I think it should be written in a more straightforward fashion.

Reply:
@driusan what do you think of the latest change? I found that the attaching a zero can be replaced with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart